src/loans/index.js

```diff
@@ -39,6 +39,7 @@ function loanTotals( data ) {
   data.federalTotal = data.perkins +
                       data.staffSubsidized +
                       data.staffUnsubsidized +
+                      data.directPlus +
```

Comment:
It represents this bit from the UI:

![screen shot 2015-12-15 at 4 32 10 pm](https://cloud.githubusercontent.com/assets/212533/11824719/72854b66-a349-11e5-873b-96c803aa7993.png)

Is that the Plus loan for parents?